e2e/npm_link_package/src/main.js

```diff
@@ -15,7 +15,7 @@ console.log(c.id())
 const fp = require('@e2e/lib')
 console.log('--@e2e/lib--')
 console.log(fp.id())
-const rulesFooA = require('../external/rules_foo/foo/node_modules/@aspect-test/a')
+const rulesFooA = require('../../rules_foo/foo/node_modules/@aspect-test/a')
```

Comment:
Interesting change needed

Reply:
yea, the path here is runfiles relative for this test. no legacy external runfiles :)